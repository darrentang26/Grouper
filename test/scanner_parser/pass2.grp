type Plane_pt = {x : Int, y : Int}
type Space_pt = {x : Int, y : Int, z : Int}

let Plane_pt sq_pt1 = {x = 0, y = 0}
let Plane_pt sq_pt2 = {x = 0, y = 5}
let Plane_pt sq_pt3 = {x = 5, y = 0}
and Plane_pt sq_pt4 = {x = 5, y = 5}
and Space_pt tip1 = {x = 0, y = 2, z = 10}
and Space_pt tip2 = {x = 5, y = 5, z = 10} in
    sq_pt1.x + sq_pt2.y + sq_pt3.x + sq_pt4.y - tip1.x * tip2.z