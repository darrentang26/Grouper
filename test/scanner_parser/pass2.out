type Space_pt = {x : Int, y : Int, z : Int}
type Plane_pt = {x : Int, y : Int}
let Plane_pt sq_pt1 = {x = 0, y = 0} in
let Plane_pt sq_pt2 = {x = 0, y = 5} in
let Plane_pt sq_pt3 = {x = 5, y = 0} in
let Plane_pt sq_pt4 = {x = 5, y = 5} in
let Space_pt tip1 = {x = 0, y = 2, z = 10} in
let Space_pt tip2 = {x = 5, y = 5, z = 10} in
sq_pt1.x + sq_pt2.y + sq_pt3.x + sq_pt4.y - tip1.x * tip2.z
